src/coreComponents/fileIO/doc/InputXMLFiles.rst

```diff
@@ -235,16 +235,51 @@ They also support any number of command-line parameter overrides (e.g. `geosx_pr
 Included Files
 ------------------------------
 
-The xml preprocessor will merge files indicated via the Included block.
-The merging approach is applied resursively, allowing children to include their own files.
-Any potential conflits are handled via the following scheme:
+Both the XML preprocessor and GEOSX executable itself provide the capability to build complex
+multi-file input decks by including XML files into other XML files.
+
+The files to be included are listed via the `<Included>` block. There maybe any number of such blocks.
+Each block contains a list of `<File name="...">` tags, each indicating a file to include.
+The `name` attribute must contain either an absolute or a relative path to the included file.
+If the path is relative, it is treated as relative to the location of the referring file.
+Included files may also contain includes of their own, i.e. it is possible to have `a.xml` include `b.xml`
```

Comment:
```suggestion
Included files may also contain includes of their own, i.e. it is possible to have `a.xml` includes `b.xml`
```

Reply:
Here I did intend the grammar as written, as in "to have somebody do something"